recipe/meta.yaml

```diff
@@ -47,3 +45,6 @@ about:
 extra:
   recipe-maintainers:
     - nicoddemus
+  skip-lints:
+    - python_build_tool_in_run
+    - missing_wheel
```

Comment:
I would remove this. We should fix the linter so that `missing_wheel` is not raised when using other build backends (hatchling, poetry, pdm, scikit-build-core, etc).